web/src/ResourceStatusSummary.test.tsx

```diff
@@ -102,3 +142,85 @@ it("does NOT link to warning and unhealthy resources when `linkToLogFilters` is
 
   expect(screen.queryByRole("link")).toBeNull()
 })
+
+describe("document metadata based on resource statuses", () => {
+  test.each<
+    [
+      caseName: string,
+      args: {
+        counts: StatusCounts
+        isSnapshot: boolean
+        isSocketConnected: boolean
+      },
+      title: string,
+      faviconColor: string
+    ]
+  >([
+    [
+      "no socket connection",
+      {
+        counts: testCountsUnhealthy,
+        isSnapshot: false,
+        isSocketConnected: false,
+      },
+      "Disconnected ┊ Tilt",
+      "gray",
+    ],
+    [
+      "viewing a snapshot",
+      {
+        counts: testCountsUnhealthy,
+        isSnapshot: true,
+        isSocketConnected: false,
+      },
+      "Snapshot: ✖︎ 2 ┊ Tilt",
+      "red",
+    ],
+    [
+      "some resources are unhealthy",
+      {
+        counts: testCountsUnhealthy,
+        isSnapshot: false,
+        isSocketConnected: true,
+      },
+      "✖︎ 2 ┊ Tilt",
+      "red",
+    ],
+    [
+      "all enabled resources are healthy",
+      { counts: testCountsHealthy, isSnapshot: false, isSocketConnected: true },
+      "✔︎ 5/5 ┊ Tilt",
+      "green",
+    ],
+    [
+      "some resources are pending",
+      { counts: testCountsPending, isSnapshot: false, isSocketConnected: true },
+      "… 3/5 ┊ Tilt",
+      "gray",
+    ],
+    [
+      "all resources are disabled",
+      {
+        counts: testCountsAllDisabled,
+        isSnapshot: false,
+        isSocketConnected: true,
+      },
+      "… 0/0 ┊ Tilt",
+      "gray",
```

Comment:
outside the scope of this pr, but i'll mention in case anyone has better suggestions! the "all resources are disabled" document title could probably be improved, so it doesn't seem like this is a "pending" / "disconnected" state.

Reply:
Perhaps `⃠`? (`&nbsp;&#x20E0;`)

Alternatively, a gray `✔︎` with `0/0` seems reasonable?

Reply:
good suggestion! i like `✔︎ 0/0`.